plugin/cache/item.go

```diff
@@ -9,15 +9,13 @@ import (
 
 type item struct {
 	Rcode              int
-	AuthenticatedData  bool
-	RecursionAvailable bool
 	Answer             []dns.RR
 	Ns                 []dns.RR
 	Extra              []dns.RR
-
-	origTTL uint32
-	stored  time.Time
-
+	AuthenticatedData  bool
```

Comment:
unrelated change and these where grouped with the other msg related fields (i.e. there was some significance in that)